
package org.esupportail.pstagedata.remote;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.7.0
 * 2012-12-04T12:58:47.966+01:00
 * Generated source version: 2.7.0
 */

@WebFault(name = "PersonalAlreadyExistingForCentreException", targetNamespace = "http://remote.pstagedata.esupportail.org/")
public class PersonalAlreadyExistingForCentreException_Exception extends Exception {
    
    private org.esupportail.pstagedata.remote.PersonalAlreadyExistingForCentreException personalAlreadyExistingForCentreException;

    public PersonalAlreadyExistingForCentreException_Exception() {
        super();
    }
    
    public PersonalAlreadyExistingForCentreException_Exception(String message) {
        super(message);
    }
    
    public PersonalAlreadyExistingForCentreException_Exception(String message, Throwable cause) {
        super(message, cause);
    }

    public PersonalAlreadyExistingForCentreException_Exception(String message, org.esupportail.pstagedata.remote.PersonalAlreadyExistingForCentreException personalAlreadyExistingForCentreException) {
        super(message);
        this.personalAlreadyExistingForCentreException = personalAlreadyExistingForCentreException;
    }

    public PersonalAlreadyExistingForCentreException_Exception(String message, org.esupportail.pstagedata.remote.PersonalAlreadyExistingForCentreException personalAlreadyExistingForCentreException, Throwable cause) {
        super(message, cause);
        this.personalAlreadyExistingForCentreException = personalAlreadyExistingForCentreException;
    }

    public org.esupportail.pstagedata.remote.PersonalAlreadyExistingForCentreException getFaultInfo() {
        return this.personalAlreadyExistingForCentreException;
    }
}
